lte/gateway/c/core/oai/tasks/ngap/ngap_state_converter.cpp

```diff
@@ -24,26 +24,26 @@ using namespace std;
 using namespace magma::lte;
 using namespace magma::lte::oai;
 
-using magma::lte::oai::Ngap_UeDescription;
```

Comment:
What is the plan for those `Ngap` to `S1ap` protobuf re-usage? Across this PR there are multiple changes that removes the usage of `Ngap` protos, do we have other usages for this? On this PR I do not see the need to keep them, do you plan to clean those up?

If so, please include these changes on this PR or as part of a separate PR and link that to this one.

Reply:
- s1ap_state.proto and ngap_state.proto are same, replaced s1ap_state.proto for ngap stateless support.
- ngap_state.proto file removed from repo and included in this PR